ui/lib/core/addon/helpers/sync-destinations.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { helper as buildHelper } from '@ember/component/helper';
+
+const SYNC_DESTINATIONS = [
+  {
+    displayName: 'AWS Secrets Manager',
+    type: 'aws-sm',
+    icon: 'aws-color',
```

Comment:
Decided against adding a `category: 'secrets'` key. If we extend lifecycle management to include certificates then we can update the variable name here or add a key, depending on what we think will be best